defaults/config.js

```diff
@@ -506,4 +506,32 @@ module.exports = {
 		//
 		raw: false,
 	},
+
+	// Search settings
+	//
+	// @type     object
+	// @default  {}
+	//
+	search: {
+		// Enables indexing for lobby messages.
+		//
+		// @type     boolean
+		// @default  false
+		//
+		indexLobby: false,
+
+		// Enables indexing only for messages (skip everything else) from channels.
+		//
+		// @type     boolean
+		// @default  false
+		//
+		indexOnlyChanMessage: true,
```

Comment:
I feel like neither of these `index` options are necessary. Just store everything, we would also eventually want to reload messages from sqlite when restarting the server.